packages-web/badge-button/package.json

```diff
@@ -16,6 +16,16 @@
     "developmentPort": 3000,
     "testProjectId": "52d48d8a-38ba-44c6-b297-1a0216c10d28",
     "testProjects": [
+      {
+        "revision": 151,
```

Comment:
I don't see this revision being used in `e2e.js`. Does this mean that we are missing something there? Or vice versa - this is extra information we don't need?

Reply:
`npm run svncheckout` use the revision

Reply:
but we don't use it in our automation flow, what's the point of keeping it? Or is it used in internal automation?

Reply:
Removed :)